Toggl.Daneel/ViewControllers/LoginViewController.cs

```diff
@@ -41,226 +32,233 @@ public override void ViewDidLoad()
+                .DisposedBy(DisposeBag);
 
-            UIColor loginButtonTintColor(bool hasError)
-                => hasError ? UIColor.White : UIColor.Black;
+            ViewModel.ClearEmailScreenError
+                .Select(_ => string.Empty)
+                .Subscribe(LoginWithEmailErrorLabel.Rx().Text())
+                .DisposedBy(DisposeBag);
 
-            UIColor loginButtonTitleColor(bool enabled) => enabled
-                ? Color.Login.EnabledButtonColor.ToNativeColor()
-                : Color.Login.DisabledButtonColor.ToNativeColor();
-        }
+            ViewModel.EmailRelay
+                .Subscribe(LoginWithEmailTextField.Rx().TextObserver())
+                .DisposedBy(DisposeBag);
 
-        public override void ViewWillAppear(bool animated)
-        {
-            base.ViewWillAppear(animated);
-            NavigationController?.SetNavigationBarHidden(true, true);
-        }
+            ViewModel.PasswordRelay
+                .Subscribe(PasswordTextField.Rx().TextObserver())
+                .DisposedBy(DisposeBag);
 
-        public override void ViewWillDisappear(bool animated)
-        {
-            base.ViewWillDisappear(animated);
-            NavigationController?.SetNavigationBarHidden(false, true);
-        }
+            ViewModel.LoginWithEmail.Errors
+                .Select(e => e.Message)
+                .Subscribe(LoginWithEmailErrorLabel.Rx().Text())
+                .DisposedBy(DisposeBag);
 
-        public override void ViewDidLayoutSubviews()
-        {
-            base.ViewDidLayoutSubviews();
+            ViewModel.IsLoggingIn
+                .Subscribe(ActivityIndicator.Rx().IsVisibleWithFade())
+                .DisposedBy(DisposeBag);
 
-            if (View.Frame.Height > iPhoneSeScreenHeight && !keyboardIsOpen)
-                TopConstraint.Constant = topConstraintForBiggerScreens;
+            ViewModel.IsLoggingIn
+                .Subscribe(UIApplication.SharedApplication.Rx().NetworkActivityIndicatorVisible())
+                .DisposedBy(DisposeBag);
 
-            SignupCard.SetupBottomCard();
-            GoogleLoginButton.SetupGoogleButton();
-        }
+            ViewModel.IsLoggingIn.Select(loginButtonTitle)
+                .Subscribe(LoginButton.Rx().AnimatedTitle())
+                .DisposedBy(DisposeBag);
 
-        public override void ViewDidAppear(bool animated)
-        {
-            base.ViewDidAppear(animated);
-            ActivityIndicator.Alpha = 0;
-            ActivityIndicator.StartSpinning();
-            PasswordTextField.ResignFirstResponder();
-        }
+            ViewModel.IsInSecondScreen
+                .Invert()
+                .Do(isGoingBack =>
+                {
+                    if (isGoingBack)
+                    {
+                        LoginWithEmailTextField.BecomeFirstResponder();
+                    }
+                })
+                .Subscribe(FirstScreenWrapperView.Rx().AnimatedIsVisible());
```

Comment:
Missing `DisposedBy` here.
```suggestion
                .Subscribe(FirstScreenWrapperView.Rx().AnimatedIsVisible())
                .DisposedBy(DisposeBag);
```